Documentation/sound/designs/timestamping.rst

```diff
@@ -143,7 +143,7 @@ timestamp shows when the information is put together by the driver
 before returning from the ``STATUS`` and ``STATUS_EXT`` ioctl. in most cases
 this driver_timestamp will be identical to the regular system tstamp.
 
-Examples of typestamping with HDaudio:
+Examples of typestamping with HDAudio:
```

Comment:
mmh, I had to re-read the commit message about 5 times or more... It really says "HDAudio and HDAudio" doesn't it? :-) Same in the next patch

Reply:
Software autocorrect or finger autocorrect? 😂

Reply:
and I didn't even see the timestamping/typestamping confusion.
I need more coffee.